{{cookiecutter.python_name}}/.yarnrc.yml

```diff
@@ -0,0 +1,11 @@
+enableImmutableInstalls: false
+
+enableInlineBuilds: false
+
+enableTelemetry: false
+
+httpTimeout: 60000
+
+nodeLinker: node-modules
+
+npmRegistryServer: "https://registry.yarnpkg.com"
```

Comment:
I would only specify the linker here.

Cc @brichet what do you think?

Reply:
The linker is the only non default value, so it should be the only required field in the file.
And it can probably be cleaned in Jupyterlab too. I'll take a look.